project_national/sdr_upgrades_tmy3.yml

```diff
@@ -0,0 +1,1149 @@
+schema_version: '0.4'
+os_version: 3.9.0
+os_sha: c77fbb9569
+buildstock_directory: ../ # Relative to this file or absolute
+project_directory: project_national # Relative to buildstock_directory
+output_directory: sdr_upgrades_tmy3
+weather_files_url: https://data.nrel.gov/system/files/156/Buildstock_TMY3_FIPS-1678817889.zip
+
+# Baseline number of units
+baseline:
+  n_buildings_represented: 139647020 # American Community Survey 2021 5-year, B25001, does not include territories
+
+# Sampler Type and number of data points per baseline or upgrade
+sampler:
+  type: residential_quota
+  args:
+    n_datapoints: 2 # For CI
+    # n_datapoints: 550000 # For production run
+
+# NREL High Performance Computing machine configuration
+# Not used if running buildstock_local (local tests and CI)
+kestrel:
+  n_jobs: 600
+  minutes_per_sim: 10
+  account: <account>
+  postprocessing:
+    time: 800
+    n_workers: 20
+    n_procs: 20
+  sampling:
+    time: 60
+
+# Post proccessing and Amazon Web Services
+postprocessing:
+  # Post simulation partitioning
+  keep_individual_timeseries: true
+  partition_columns:
+    - State
+    - County
+  # AWS upload bucket and athena setup (uncomment during production and testing runs)
+  # aws:
+  #  region_name: us-west-2
+  #  s3:
+  #    bucket: sdr_resstock
+  #    prefix: production_runs
+  #  athena:
+  #    glue_service_role: service-role/AWSGlueServiceRole-default
+  #    database_name: sdr_resstock
+  #    max_crawling_time: 1200
+
+# Workflow generator
+workflow_generator:
+  # ResStock HPXML workflow generator
+  type: residential_hpxml
+  version: 2024.07.20
+
+  args:
+    # Annual simulation and timestep
+    build_existing_model:
+      simulation_control_timestep: 15
```

Comment:
Is it necessary to have 15 instead of 60 here?

Reply:
We might need to for CI to get a couple more datapoints simulated. 15 min is what we run for the dataset publication.